CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` and `Removed`.
 
 ## [Unreleased]
+### Added
+- Forest Night theme
```

Comment:
Since we aren't currently shipping themes with the binary we shouldn't add an entry to the `CHANGELOG.md`.